src/helm/ralph/templates/ingress.yaml

```diff
@@ -1,44 +1,61 @@
 {{- if .Values.ingress.enabled -}}
+{{- $fullName := include "ralph.fullname" . -}}
+{{- $svcPort := .Values.service.port -}}
+{{- if and .Values.ingress.className (not (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion)) }}
+  {{- if not (hasKey .Values.ingress.annotations "kubernetes.io/ingress.class") }}
+  {{- $_ := set .Values.ingress.annotations "kubernetes.io/ingress.class" .Values.ingress.className}}
+  {{- end }}
+{{- end }}
+{{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion -}}
 apiVersion: networking.k8s.io/v1
+{{- else if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+apiVersion: networking.k8s.io/v1beta1
+{{- else -}}
+apiVersion: extensions/v1beta1
+{{- end }}
```

Comment:
@jmaupetit we have already talk about support deprecated version. But I don't remember your decision.

Reply:
You would advise against supporting depredated k8s version?

Reply:
Yes, but it's my opinion.

Reply:
Leaving as is for now!